### PR TITLE
Fix/various small fixes

### PR DESCRIPTION
**Pull request by paulmskim**
https://central.tri.be/issues/130296

### CODE REVIEW

File: src/views/v2/month/calendar-header.php [hunk 1]
```diff
@@ -18,7 +18,7 @@
 ?>
 <header role="rowgroup">
 
-	<h2 class="tribe-common-a11y-visual-hide" id="tribe-calendar-header"><?php printf( esc_html__( 'Calendar of %s', 'the-events-calendar' ), tribe_get_event_label_plural() ); ?></h2>
+	<h2 class="tribe-common-a11y-visual-hide" id="tribe-events-calendar-header"><?php printf( esc_html__( 'Calendar of %s', 'the-events-calendar' ), tribe_get_event_label_plural() ); ?></h2>
```
Comment (tr1b0t) [inline src/views/v2/month/calendar-header.php]: :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: e203e49d252f935ff4fa4354ab11d2de</sub>

File: src/views/v2/month/calendar-header.php [hunk 2]
```diff
@@ -16,18 +16,18 @@
 $days_of_week = tribe_events_get_days_of_week();
 global $wp_locale;
 ?>
-<header role="rowgroup">
+<header class="tribe-events-calendar-month__header" role="rowgroup">
 
-	<h2 class="tribe-common-a11y-visual-hide" id="tribe-calendar-header"><?php printf( esc_html__( 'Calendar of %s', 'the-events-calendar' ), tribe_get_event_label_plural() ); ?></h2>
+	<h2 class="tribe-common-a11y-visual-hide" id="tribe-events-calendar-header"><?php printf( esc_html__( 'Calendar of %s', 'the-events-calendar' ), tribe_get_event_label_plural() ); ?></h2>
```
Comment (tr1b0t) [inline src/views/v2/month/calendar-header.php]: :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 0d1111cfbb897e1c9b67a54b97cfa1c2</sub>

File: src/resources/postcss/components/_view-selector.pcss [hunk 3]
```diff
@@ -256,9 +256,12 @@
 		&:hover,
 		&:focus {
 
+			.tribe-events-c-view-selector__list-item-icon {
+				opacity: var(--opacity-hover);
```
Comment (juanfra) [inline src/resources/postcss/components/_view-selector.pcss]: Do we need to update the var name here?

Comment (paulmskim) [reply src/resources/postcss/components/_view-selector.pcss]: see the latest commit, i think you're reviewing on an old diff.

Comment (juanfra) [reply src/resources/postcss/components/_view-selector.pcss]: Yep, I was haha! :)

**Pull request merged**